fuzztest.bazelrc

```diff
@@ -20,12 +21,19 @@ build:fuzztest-common --copt=-UNDEBUG
 # Enable libc++ assertions.
 # See https://libcxx.llvm.org/UsingLibcxx.html#enabling-the-safe-libc-mode
 build:fuzztest-common --copt=-D_LIBCPP_ENABLE_ASSERTIONS=1
+
 ### ASan (Address Sanitizer) build configuration.
 #
 # Use with: --config=asan
 
 build:asan --linkopt=-fsanitize=address
 build:asan --copt=-fsanitize=address
+
```

Comment:
FYI, this file changed because of the updated fuzztest which generates a new bazelrc include.